openwisp_radius/tests/test_admin.py

```diff
@@ -220,7 +220,10 @@ def test_radiuscheck_change(self):
 
     def test_radiusbatch_change(self):
         obj = self._create_radius_batch(
-            name='test', strategy='prefix', prefix='test', expiration_date='1998-01-28'
+            name='test',
+            strategy='prefix',
+            prefix='test-prefix4',
```

Comment:
Change 1: I had to change the `prefix` for all the tests so they have unique prefixes and unique files created because for finding which file was created from which testcase, I could only perform on hit-and-trail which was a waste of time.